### PR TITLE
Add Java 11 testing in Azure

### PR DESCRIPTION
**Pull request by chrisdennis**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -20,8 +20,13 @@
...
       name: terracotta/terracotta
 
 jobs:
-- template: build-templates/gradle-common.yml@templates  
+- template: build-templates/gradle-common.yml@templates
+  parameters:
+    jdkVersion: '1.8'
+- template: build-templates/gradle-common.yml@templates
+  parameters:
+    jdkVersion: '11'
```
Comment (akomakom) [inline azure-pipelines.yml]: It's 1.11, see toolchains.xml in [any build](https://dev.azure.com/TerracottaCI/terracotta/_build/results?buildId=2771)

File: azure-pipelines.yml [hunk 2]
```diff
@@ -20,8 +20,13 @@
...
     - repository: templates
-      type: git   
+      type: git
       name: terracotta/terracotta
 
 jobs:
-- template: build-templates/gradle-common.yml@templates  
+- template: build-templates/gradle-common.yml@templates
+  parameters:
+    jdkVersion: '1.8'
```
Comment (akomakom) [inline azure-pipelines.yml]: try 
`jobName: '1.8 build' `
here

**Comment by chrisdennis** [comment]
I'm happy to merge this without Windows Java 11 support if that makes things easier for you @akomakom

**Comment by akomakom** [comment]
> I'm happy to merge this without Windows Java 11 support if that makes things easier for you @akomakom

windows support is in place for gradle - both gradle.properties and azure steps have `javaNHome`

**Pull request merged**